File: rb/utility/make_context.rb
# PlaystationStoreApi2 SDK utility: make_context
require_relative '../core/context'
module PlaystationStoreApi2Utilities
  MakeContext = ->(ctxmap, basectx) {
    PlaystationStoreApi2Context.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// PlaystationStoreApi2 SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class PlaystationStoreApi2MakeContext
{
    public static function call(array $ctxmap, ?PlaystationStoreApi2Context $basectx): PlaystationStoreApi2Context
    {
        return new PlaystationStoreApi2Context($ctxmap, $basectx);
    }
}
